R/flatten.R

```diff
@@ -76,8 +77,9 @@ cdm_squash_to_tbl <- function(dm, start, ..., join = left_join) {
 
 cdm_flatten_to_tbl_impl <- function(dm, start, ..., join, join_name, squash) {
   check_correct_input(dm, start)
-  list_of_pts <- as.character(enexprs(...))
-  check_correct_input(dm, list_of_pts)
+  vars <- setdiff(tidyselect_table_names(dm), start)
+  list_of_pts <- tidyselect::vars_select(vars, ...)
```

Comment:
What's the wording of the error message when the table isn't found?

Reply:
`Error in is_character(x) : object 'dim_5' not found`...

better might be a `tryCatch()` with `abort_table_not_in_dm(setdiff(tables, src_tbls(dm)), dm))` in case of an error, but it seems non-trivial to find the wrong columns in case the user uses deselection or select helpers

Reply:
An alternative would be to extend the {tidyselect} error message (also in a `tryCatch()`), adding which tables are available.

Reply:
Implemented in 4fc8eb8